engines/gob/detection/tables_adibou2.h

```diff
@@ -114,7 +117,8 @@
 	{
 		"adibou2",
 		"ADIBOU 2.13s Environnement",
-		AD_ENTRY1s("intro.stk", "092707829555f27706920e4cacf1fada", 8737958),
+		AD_ENTRY2s("intro.stk", "092707829555f27706920e4cacf1fada", 8737958,
+				"BECBD213.CD1",	"bc828c320908a5eaa349956d396bd8e1", 8),
```

Comment:
This one is correct, since the MD5 hash of intro.stk is not unique, so adding another file for correct detection is the correct approach

Reply:
@BJNFNE please check this entry. Since it's the french version the .CD1 file should be BECBF213.CD1

Reply:
can you provide me the md5 hash and filesize or is it the same?

Reply:
> can you provide me the md5 hash and filesize or is it the same?

You changed the entry so I thought you had the game? Anyway the hash/filesize is the same (which is not a great thing, it would have been better to use a more meaningful file instead of this one which is identical in all releases and just contains a string)